cea/utilities/reporting.py

```diff
@@ -40,18 +38,25 @@ def full_report_to_xls(tsd, output_folder, basename):
     writer.close()
 
     # quick visualization
-    quick_visualization_tsd(tsd, output_folder, basename)
+    # UNCOMMENT THE NEXT LINE TO PRODUCE QUICK VISUALIZATIONS WITH PLOTLY OF THE DEMAND CALCULATION IN BUILDINGS
+    # quick_visualization_tsd(tsd, output_folder, basename)
```

Comment:
please don't hide functionality in comments. make this an option to the script.

Reply:
Again, just copying what I saw above.
I will delete the code. And keep it somewhere for myself.
I think it's useful for testing and understanding what's happening.
It's not really for researchers, it's more for developers.

Reply:
the debug flag could help?